compiler/noirc_evaluator/src/ssa/opt/inline_simple_functions.rs

```diff
@@ -124,14 +129,22 @@ mod test {
           b0(v0: Field):
             v1 = call f1(v0) -> Field
             v2 = call f1(v0) -> Field
-            v3 = add v1, v2
-            return v3
+            v3 = call f1(v0) -> Field
+            v4 = call f1(v0) -> Field
+            v5 = call f1(v0) -> Field
+            v6 = call f1(v0) -> Field
+            v7 = call f1(v0) -> Field
+            v8 = call f1(v0) -> Field
+            v9 = call f1(v0) -> Field
+            v10 = call f1(v0) -> Field
```

Comment:
Can you leave a comment about why these unused calls have been added? Previously it was 1 per argument of `add`, which was more intuitive. For example it looks like they will all be removed from `bar` anyway, so, does matter how many were added?

Reply:
Right, I just saw they were eliminated from `f2` completely. I guess they could be any instruction whatsoever, it was just slightly suspect that it's a call to `f1` itself, like, not clear if we're trying to express that "if you try to inline f1 more than 10 times then X happens", rather than "if f2 has more than 10 instructions it's not inlined into f0"